presets/4.3/vtx/Freq_change_with_3-6pos_SW.txt

```diff
@@ -0,0 +1,101 @@
+#$ TITLE: Transmitter VTX freq change with 3pos/6pos Switch
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, vtx table, Japan, Japanese, JP, JA, Jumper-T16, TX16S
+#$ AUTHOR: ToshihiroMakuuchi
+#$ DESCRIPTION: ----------
+#$ DESCRIPTION: Use the 3pos and 6pos switches on the transmitter to set the VTX frequency to change.
+#$ DESCRIPTION: 
+#$ DESCRIPTION: -- Amateur Radio License in Japan --
+#$ DESCRIPTION: Radio communications most commonly available include amateur, civil, and personal radio communications.
+#$ DESCRIPTION: Amateur radio communications are authorized only for those who pass a national license examination or who complete JARD & other official training course and acquire the prescribed radio operator certificate.
```

Comment:
I think this description is lacking information about how exactly this preset going to work.
As well as option names are not clear. I do not understand what that means for example:
`vtx_power=1&2`
i am pretty sure most user won't understand that too.

Reply:
I understand. I will rewrite the information that three frequencies at the same time within the scope of the Japanese band plan can be switched immediately using a switch on the transmitter.
We will also reconsider the name used for the option. Please wait a moment.